omero/institution/getting-started.html

```diff
@@ -9,8 +9,108 @@
+            </div>
+            <div>
+                <h5>Selected OMERO features</h5>
+                <ul class="dev-ops-links">
+                    <li><a href="{{ site.baseurl }}/omero/features/view/" target="_blank">Access and view your images remotely via internet</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/features/share/" target="_blank">Cooperate on images with your colleagues</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/figure/" target="_blank">Create figures for scientific papers or colleagues</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/features/publish/" target="_blank">Publish your original image data</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/features/analyze/" target="_blank">Analyze the images in OMERO via API, using established third party image analysis software.</a></li>
+                </ul>
+                <h5>Read and watch about OMERO to get a better idea.</h5>
+                <ul class="dev-ops-links">
+                    <li><a href="{{ site.baseurl }}/omero/institution/" target="_blank">Further read on advantages of OMERO for your institution</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/features/index.html" target="_blank">List of OMERO features</a></li>
+                    <li><a href="https://omero-guides.readthedocs.io/en/latest/index.html" target="_blank">OMERO guides</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/index.html" target="_blank">OMERO documentation</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/apps/" target="_blank">List of OMERO Apps</a></li>
+                    <li><a href="https://www.youtube.com/channel/UCyySB9ZzNi8aBGYqcxSrauQ/videos" target="_blank">Watch videos on openmicroscopy YouTube channel</a></li>
+                    <li><a href="{{ site.baseurl }}/training/" target="_blank">See how to get a training on OMERO or organize one yourself</a></li>
+                </ul>
+            </div>
+        </div>
+        <!-- end What is needed -->
+        <a id="what-is-needed"></a>
+        <hr class="whitespace">
+        <div class="row column text-center">
+            <h3>What is needed to host OMERO?</h3>
+        </div>
+        <hr>
+        <div class="row">
+            <div class="medium">
+                <p>The resources needed for setting up, keeping and maintaining OMERO vary according to your image data sizes and quality, and the infrastructure in your institution. Nevertheless, some general hints are given here as points which you should think about and implement in your concrete case.</p>
+                <p>You can benefit from asking your colleagues at Institutions which are already running OMERO. For that, we also list some contacts to regional groups and list of selected institutions which run OMERO in the <a href="#contacts">Key OMERO Contacts</a> section.</p>
```

Comment:
Institutions doesn't need to be upper case.

Reply:
fixed in https://github.com/ome/www.openmicroscopy.org/pull/462/commits/b34402411fec182b880b40934783bd667f93945f